phc-sf-spec.md

```diff
@@ -279,6 +286,8 @@ For Argon2, the following is specified:
 
   - The identifier for Argon2ds is `argon2ds`.
 
+  - The versions are: [16, 19].
```

Comment:
It would be clearer to use `0x10, 0x13` or `16 (for Argon2 1.0), 19 (for Argon2 1.3)`.

Reply:
Yes it makes sense.
Actually I think we should completely remove the **Argon2 Encoding** section.
For me it does not any make sense have it here.